peripherals/sht2x/Kconfig

```diff
@@ -1,14 +1,18 @@
 
 # Kconfig file for package sht2x
 menuconfig PKG_USING_SHT2X
-    bool "a sht2x package for rt-thread"
+    bool "digital humidity and temperature sensor sht2x driver library"
     default n
 
 if PKG_USING_SHT2X
 
     config PKG_SHT2X_PATH
         string
         default "/packages/peripherals/sht2x"
+		
+	config SHT20_USING_SOFT_FILTER
+        bool "Enable support filter function"
+        default n
```

Comment:
```
config SHT20_USING_SOFT_FILTER
        bool "Enable average filter by software"
        default n
        help "It will automatic sample sensor's data by average filter thread."
```